specification/network/resource-manager/Microsoft.Network/stable/2019-07-01/virtualNetworkGateway.json

```diff
@@ -3056,6 +3302,24 @@
         }
       },
       "description": "VPN client connection health detail."
+    },
+    "VpnPacketCaptureStopParameters": {
+      "properties": {
+        "sasUrl": {
+          "type": "string",
+          "description": "SAS url for packet capture on VNG."
+        }
+      },
+      "description": "Stop packet capture parameters."
+    },
+    "VpnPacketCaptureStartParameters": {
+      "properties": {
+        "filterData": {
```

Comment:
Why is this modeled as a JSON string instead of a rich object?  Where does the user get this filterData string from and how do they figure out the format?

Reply:
This is modeled as a JSON to have more flexibility in publishing changes to filtering data. We are going to publish this JSON schema in public domain for the customer to refer to.